### ISSUE TITLE
install NodeJS plugin in PyCharm

### ISSUE THREAD

**Issue by stevepiercy** [pauleveritt/pauleveritt.github.io#2]
This step is missing a critical step.
http://www.pauleveritt.org/articles/pylyglot/hello_node/

Install and enable the NodeJS plugin. The plugin is not bundled with PyCharm, but it can be installed from the JetBrains plugin repository as described in Installing, Updating and Uninstalling Repository Plugins and Enabling and Disabling Plugins.

I reckon since you had already installed it, you overlooked it in the narrative.

**Comment by pauleveritt** [comment]
You’re right, thanks. Before making this generally available, I’ll get this fixed.

—Paul
